config/deploy.rb

```diff
@@ -58,6 +59,7 @@
     end
   end
 
+  #TODO FIX: remove password from wp core install command
   desc "Wp Core Install"
```

Comment:
Yes, using cap I've tested multiple deploys. It runs successfully and just returns "wordpress already installed" when that wp command is run. Yea makes more sense to move it to Ansible. I'll sort that out.